app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -286,10 +286,9 @@ class ZimManageViewModel @Inject constructor(
     val booksFromDao = books().asFlowable()
     val networkLibrary = PublishProcessor.create<LibraryNetworkEntity>()
     val languages = languageDao.languages().asFlowable()
-    return arrayOf(
+    return arrayOf<Disposable>(
```

Comment:
We don't need to define it explicitly, we already defined the type of return type of the method.